apis/python/src/tiledbsoma/pytiledbsoma.cc

```diff
@@ -120,6 +120,34 @@ PYBIND11_MODULE(pytiledbsoma, m) {
         },
         "Print TileDB internal statistics. Lifecycle: experimental.");
 
+    m.def(
```

Comment:
recommend tests for these functions (directly called).  In particular, test them for garbage inputs - do they raise correct errors, etc.

Reply:
I added basic tests for the PyBind functions, but I only added the tests for non-SOMA objects to the C++ layer since we don't currently install TileDB-Py.